ext/SciMLBaseZygoteExt.jl

```diff
@@ -299,4 +300,23 @@ end
     ∇responsible_map(__context__, f, args...)
 end
 
+@_adjoint_keepthunks function Zygote.literal_getfield(x::ODEProblem, ::Val{f}) where f
+  val = getfield(x, f)
+  function back(Δ)
+    # error()
+    Zygote.accum_param(__context__, val, Δ) === nothing && return
+    if isimmutable(x)
+      error()
+      dx = (; Zygote.nt_nothing(x)..., pair(Val(f), Δ, x)...)
+      (_project(x, dx), nothing)
+    else
+      dx = Zygote.grad_mut(__context__, x)
+      dx[] = (; dx[]..., pair(Val(f), Zygote.accum(getfield(dx[], f), Δ))...)
+      return (dx,nothing)
```

Comment:
What's the point of this block if the first line is `error()`?

Reply:
Removed in [a03bb02](https://github.com/SciML/SciMLBase.jl/pull/1036/commits/a03bb02cc059c88aa27f04aafe6fa00893888585)